core/toolbox/separator.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A separator used for separating toolbox categories.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ * @author maribethb@google.com (Maribeth Bottorff)
+ */
+'use strict';
+
+goog.provide('Blockly.ToolboxSeparator');
+
+goog.require('Blockly.ToolboxItem');
+
+
+/**
+ * Class for a toolbox separator. This is the thin visual line that appears on
+ * the toolbox. This item is not interactable.
+ * @param {!Blockly.utils.toolbox.Separator} toolboxSeparatorDef The information
+ *     needed to create a separator.
+ * @param {!Blockly.IToolbox} toolbox The parent toolbox for the separator.
+ * @constructor
+ * @extends {Blockly.ToolboxItem}
+ */
+Blockly.ToolboxSeparator = function(toolboxSeparatorDef, toolbox) {
+
+  Blockly.ToolboxSeparator.superClass_.constructor.call(
+      this, toolboxSeparatorDef, toolbox);
+  /**
+   * The config for all the separator classes.
+   * @type {!Blockly.ToolboxSeparator.ClassConfig}
+   * @protected
+   */
+  this.classConfig_ = {
```

Comment:
I think some confusion I'm having is with usage of the term classes. Class could be either referring to JavaScript class or css class, after looking a bit longer it seems like this is about css. Maybe mentioning css in the JsDoc would make this clearer.

Reply:
I added what I think is a better comment, but would it also help to name these cssConfig_ instead of classConfig_?

Reply:
I think `cssConfig_` could also help. I do like the jsdoc comment changes.